beagle_cli

```diff
@@ -175,6 +184,22 @@ def _check_is_authenticated(config):
 
 # List commands
 
+def _get_runs_command(arguments, config):
+    run_id = arguments.get('--run-id')
+    page_size = arguments.get('--page-size')
+    params = dict()
+    if page_size:
+        params['page_size'] = page_size
+    url = urljoin(BEAGLE_ENDPOINT, API['run'])
+    if run_id:
+        url = url + run_id
+    response = requests.get(url,
+                            headers={'Authorization': 'Bearer %s' % config.token}, params=params)
+    response_json = json.dumps(response.json(), indent=4)
+    config.set('prev', None)
+    config.set('next', None)
```

Comment:
We also need this https://github.com/mskcc/beagle_cli/blob/develop/beagle_cli.py#L230 because response is paginated.